unittest/fortran/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+include(CheckGeneratorSupport)
+if(NOT CMAKE_GENERATOR_SUPPORT_FORTRAN)
+  message(STATUS "Skipping Tests for the LAMMPS Fortran Module: no Fortran support in build tool")
+  return()
+endif()
+
+include(CheckLanguage)
+check_language(Fortran)
+if(CMAKE_Fortran_COMPILER)
+  enable_language(Fortran)
+  get_filename_component(LAMMPS_FORTRAN_MODULE ${LAMMPS_SOURCE_DIR}/../fortran/lammps.f90 ABSOLUTE)
+  if(BUILD_MPI)
+    find_package(MPI REQUIRED)
+  else()
+    add_library(fmpi_stubs STATIC mpi_stubs.f90)
+    add_library(MPI::MPI_Fortran ALIAS fmpi_stubs)
```

Comment:
I recall from #1956, there was a problem with aliasing a target into the `MPI::` namespace for cmake-3.10 hence we didn't do it for `mpi_stubs` and have the `mpi_stubs` hackery e.g. in `GPU.cmake`. It might not matter here as `MPI::MPI_Fortran` has include dir, but it might also break defines.

Reply:
Compiling and testing works fine and without a problem on Ubuntu 18.04 (which has CMake 3.10). https://ci.lammps.org/blue/organizations/jenkins/dev%2Fpull_requests%2Fubuntu%2Funit_tests%2Fcmake_mpi_openmp_smallsmall_shared/detail/cmake_mpi_openmp_smallsmall_shared/813/tests

Reply:
Ok, that is great maybe we can remove the `mpi_stubs` hackery as well.